.github/ISSUE_TEMPLATE/security.md

```diff
@@ -7,4 +7,7 @@ assignees: ''
 
 ---
 
-Please for opening security issue, please contact tbd@eiger.co . For security reasons, DO NOT PUBLISH HERE INFORMATION.
+⚠️ Disclaimer: The current contracts are POC and are still not
+reviewed by security teams. We do not encourage their use in production yet.
+
+Please for opening security issue, please contact hello@eiger.co . For security reasons, DO NOT PUBLISH HERE INFORMATION.
```

Comment:
Double `please`. I would rephrase it to:
`For opening security issue, please contact hello@eiger.co `